src/index.ts

```diff
@@ -319,11 +321,7 @@ export async function install(
     external: externalPackages,
     treeshake: {moduleSideEffects: 'no-external'},
     plugins: [
-      !isStrict &&
-        rollupPluginReplace({
-          'process.env.NODE_ENV': nodeEnv ? `"${nodeEnv}"` : defaultNodeEnv,
-          'process.env.': '({}).',
-        }),
+      !isStrict && rollupPluginReplace(env! as any),
```

Comment:
Lets keep 'process.env.': '({}).', as a fallback. Can you update this to:

```
rollupPluginReplace({
  ...env,
  'process.env.': '({}).',
});
```

Getting rid of the `! as any` would be great as well

Reply:
Done